### PR TITLE
feature(admin): add route to get service configuration

### LINKED ISSUE #49: [feature] Get `/services/:service` implementation
Create GET `/services/:service` like it is specified inside https://github.com/IGNF/road2/blob/develop/documentation/apis/administration/1.0.0/api.yaml. 

- [ ] Code it (see next paragrah)
- [ ] Update doc (ex. functionalities, changelog)
- [ ] Update tests (new classes if there are; rtest for this route)
- [ ] Run tests inside docker (rtest, ctest, utest, itest)

### Code 

- Add the route inside the router of admin/1.0.0 (no need of an interface)
- Create a function getServiceConfiguration() inside administrator.js to handle the request and to get the information. Everything is in this._configuration of the administrator instance (this._configuration is the content of road2.json) : 
  - Id is in the config of the admin (aka road2.json)
  - the rest is the content of the config file of the service (aka service.json)
- Refactor function `getServicesConfigurations()` to use the new `getServiceConfiguration()` to get configuration for each service

### PR DESCRIPTION
**Pull request by benoitblanc**
Closes #49

### CODE REVIEW

**Review by lgrd — COMMENTED** [review]
That's an effective work. It works and that's cool ! I've made a lot a comment but they are on the form. There is only one comment concerning the logic inside road2.

File: src/js/administrator/administrator.js [hunk 1]
```diff
@@ -576,29 +576,68 @@ module.exports = class Administrator {
         let responses = new Array();
 
         // Pour chaque service, on récupère la configuration depuis le fichier de configuration
-        for (let i = 0; i < this._configuration.administration.services.length; i++) {
+        for (let i = 0; i < this._configuration.administration.services.length; i++) {            
+            const configuration = this.readServiceConfiguration(this._configuration.administration.services[i]);
```
[the remark this answers is not in the document]
Comment (benoitblanc) [reply src/js/administrator/administrator.js:580]: I don't agree here, you can declare a `const` object and add/remove/update its properties, no ? I do not re-assign `configuration`

File: src/js/administrator/administrator.js [hunk 2]
```diff
@@ -576,29 +576,68 @@ module.exports = class Administrator {
...
 
         return responses;
     }
 
+    /**
+     *
+     * @function
+     * @name getServiceConfiguration
+     * @description Récupération de la configuration d'un service
+     * @param {json} response - Reponse json contenant la configuration du service
```
[the remark this answers is not in the document]
Comment (benoitblanc) [reply src/js/administrator/administrator.js:596]: Actually, in `getServicesConfigurations`, we don't need any parameters

File: src/js/administrator/administrator.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lgrd) [reply src/js/administrator/administrator.js]: I revised the doc and you're right. We can change its content when it's an object. So let it like this if you want. ;)

Until now, I have used `const` to say that my variable will not change at all. In the other case, I have used `let`. I think it is a way to mark a difference.

File: src/js/administrator/administrator.js [hunk 3]
```diff
@@ -576,29 +576,68 @@ module.exports = class Administrator {
...
+     *
+     */
+
+    getServiceConfiguration(parameters) {
+
+        LOGGER.info("getServiceConfiguration...");
+
+        const service = this._configuration.administration.services.find(service => service.id == parameters.service)
+
+        if (service) {
```
[the remark this answers is not in the document]
Comment (lgrd) [reply src/js/administrator/administrator.js]: Because there is only one line after and it is a return, we can avoid a try/catch I think.

**Review by lgrd — APPROVED** [review]

**Pull request merged**